src/utils/validateIdentifierInputs.ts

```diff
@@ -24,21 +24,30 @@ export const validateIdentifierInputs = (
     validationRegex,
     textInputType,
     isOptional,
+    isTextInputDisabled,
   } of identifierInputs) {
     if (isOptional && !value) {
       return true;
     }
 
+    const invalidIdentifierInputErr = isTextInputDisabled
```

Comment:
should we have the variable naming consistent as the one [above](https://github.com/rationally-app/mobile-application/pull/558/files#diff-5ca03bfb3c7f0e9145da804ad4a282a2464f82ce08b8bc14f2ee75a0d9ae9cebR211-R213)? 

i.e.
```suggestion
    const invalidIdentifierInputErrMsg = isTextInputDisabled
```

Reply:
updated!